disdrodb/l0/check_metadata.py

```diff
@@ -146,7 +163,20 @@ def get_archive_metadata_key_value(disdrodb_dir, key, return_tuple=True):
 
 #### --------------------------------------------------------------------------.
 #### Metadata Archive Checks
-def check_archive_metadata_keys(disdrodb_dir):
+def check_archive_metadata_keys(disdrodb_dir: str) -> bool:
```

Comment:
Just a thought: 

In python, I am used to having functions (and I think most often in the repo is like this) returning True/False when the function starts with `is_valid_<archive_metadata_keys>`, while `check_<something>` usually returns the "corrected" input arguments or returns `None` (raising an error if it is the case). Also `is_<valid_something>` typically does not raise errors but returns `True/False`.

Reply:
Maybe we could have the `check_<something>` raising an error (with an optional keywords argument raise_error=True/False), and then define `is_valid_<something>`, which call  `check_<something>` inside a try/catch and returns True/False accordingly ...